sample_cond_export.txt

```diff
@@ -0,0 +1,308 @@
+class GraphModule(torch.nn.Module):
+    def forward(self, orig_arg_0, orig_arg_1, orig_arg_2):
+        arg0: b8[], arg1: f32[s0, s0], arg2: f32[s0, s0], = fx_pytree.tree_flatten_spec([orig_arg_0, orig_arg_1, orig_arg_2], self._in_spec)
+        # No stacktrace found for following nodes
+        true_graph_0 = self.true_graph_0
+        false_graph_0 = self.false_graph_0
+        
+        # File: /scratch/bahuang/work/repos/pytorch/example_control_flow.py:26, code: return cond(pred, self.true_fn, self.false_fn, [x, y])
+        cond: f32[s0, s0] = torch.ops.cond(arg0, true_graph_0, false_graph_0, [(arg1, arg2)]);  arg0 = true_graph_0 = false_graph_0 = arg1 = arg2 = None
+        return pytree.tree_unflatten([cond], self._out_spec)
+        
+    class wrapped(torch.nn.Module):
+        def forward(self, flat_args):
+            flat_args_1: f32[s0, s0], flat_args_2: f32[s0, s0], = fx_pytree.tree_flatten_spec([flat_args], self._in_spec)
+            # File: /scratch/bahuang/work/repos/pytorch/example_control_flow.py:26, code: return cond(pred, self.true_fn, self.false_fn, [x, y])
+            mul: f32[s0, s0] = torch.ops.aten.mul.Tensor(flat_args_1, flat_args_2);  flat_args_1 = flat_args_2 = None
+            return pytree.tree_unflatten([mul], self._out_spec)
+            
+    class wrapped(torch.nn.Module):
+        def forward(self, flat_args):
+            flat_args_1: f32[s0, s0], flat_args_2: f32[s0, s0], = fx_pytree.tree_flatten_spec([flat_args], self._in_spec)
+            # File: /scratch/bahuang/work/repos/pytorch/example_control_flow.py:26, code: return cond(pred, self.true_fn, self.false_fn, [x, y])
+            add: f32[s0, s0] = torch.ops.aten.add.Tensor(flat_args_1, flat_args_2);  flat_args_1 = flat_args_2 = None
+            return pytree.tree_unflatten([add], self._out_spec)
+            
+GraphModule(
+    name='GraphModule',
+    graph=Graph(
+        inputs=[
+            TensorArgument(name='arg0'),
+            TensorArgument(name='arg1'),
+            TensorArgument(name='arg2')
+        ],
+        outputs=[TensorArgument(name='cond')],
+        nodes=[
+            Node(
+                op='call_function',
+                target='<torch._ops.PyOperator object at 0x7fe67106f340>',
+                args=[
+                    Argument(as_tensor=TensorArgument(name='arg0')),
+                    Argument(
+                        as_gm=GraphModule(
```

Comment:
true_fn branch, serialized as a GraphModule, is stored as a node's argument. 

cc @avikchaudhuri